### PR TITLE
Adjust Layout for News Cells

### PR DESCRIPTION
**Pull request by ecotopian**

### CODE REVIEW

File: Client/Ecosia/UI/NewsCell.swift [hunk 1]
```diff
@@ -6,22 +6,43 @@ import Core
 import UIKit
 
 final class NewsCell: UICollectionViewCell, Themeable {
+    struct Positions: OptionSet {
+        static let top = Positions(rawValue: 1)
+        static let bottom = Positions(rawValue: 1 << 1)
```
Comment (vauxhall) [inline Client/Ecosia/UI/NewsCell.swift:11]: I feel this is a bit too complex. Wouldn't it be enough to have a struct `Positions` with 2 parameters: top, and bottom, both of them boolean?
Just a suggestion, and optional.

**Review by vauxhall — APPROVED** [review]
Looking good 👍

File: Client/Ecosia/UI/NewsCell.swift (hunk 1 above)
Comment (ecotopian) [reply Client/Ecosia/UI/NewsCell.swift:11]: I also felt like overshooting for such a simple task :D I agree that it is complicated but not complex. I needed to find a way to handle permutations of those two states. Two booleans will work also, but I find an OptionSet easier to extend. So I prefer keeping it :)

Comment (vauxhall) [reply Client/Ecosia/UI/NewsCell.swift:11]: I'm wondering if we should adjust to the current needs instead, and refactor when we get the need for extending.

**Pull request merged**